### PR TITLE
Create Classes for Food model

### PR DESCRIPTION
**Pull request by akhilvuputuri**

### CODE REVIEW

File: src/main/java/seedu/address/model/food/Food.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
+package seedu.address.model.food;
+
+import java.util.Objects;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
```
Comment (sheexiong) [inline src/main/java/seedu/address/model/food/Food.java]: I think you can import static class first then import java class, this should solve the checkstyle issue.

Comment (akhilvuputuri) [reply src/main/java/seedu/address/model/food/Food.java]: Oh yes, thanks Shee Xiong! I ran my check style and it told me the same thing too

**Comment by sheexiong** [comment]
@akhilvuputuri I think ur import order is wrong

**Review by sheexiong — APPROVED** [review]
LGTM

**Pull request merged**